src/net/sf/mpxj/primavera/PrimaveraReader.java

```diff
@@ -692,33 +863,39 @@ private void addTaskUDFValue(Task task, Row row)
                break;
             }
 
+            case BOOLEAN:
+            {
+               value = STATICTYPE_UDF_MAP.get(row.getString("udf_text"));
```

Comment:
This actually had a simple solution. I switched the default back to mapping the P6 static type indicators to Text so the default behavior stays the same as before. I just left in the support I added for parsing the indicator text as booleans and now in my specific client code I can use the pre-existing `PrimaveraXERFileReader.setFieldNamesForTaskUdfType` function to set my preference for Flag. I think that should make everyone happy. eb585e68